django/core/serializers/python.py

```diff
@@ -91,6 +92,7 @@ def Deserializer(object_list, **options):
     """
     db = options.pop('using', DEFAULT_DB_ALIAS)
     ignore = options.pop('ignorenonexistent', False)
+    handle_forward_references = options.pop('handle_forward_references', False)
```

Comment:
I've thought about this a little further.  I think this is required, since automatically handling forward references changes the behaviour of user code.

In particular, somebody might be relying on an exception being raised when they try to deserialize data that contains forward references.  The new behaviour won't raise an exception, and instead the same user code will create incomplete objects in the database.

This is because deserialization now requires two steps: firstly, we try to deserialize some data, and secondly, we have to call `save_deferred_fields` on any deserialized objects that had forward references.

Is this an argument to add some new code, to be used by `loaddata` and by user code, that removes this foot-gun?